csrc/preseg_passes/mark_aliases_prepare.cpp

```diff
@@ -107,8 +105,7 @@ void MarkAliasesPreparePass::runPass(Fusion* fusion) {
   }
 
   if (isDebugDumpEnabled(DebugDumpOption::PreSegmenterLogging)) {
-    debug() << "Fusion after MarkAliasesPreparePass:" << std::endl;
-    fusion->printMath();
+    debug() << "Fusion Transforms after MarkAliasesPreparePass:" << std::endl;
```

Comment:
How about replacing MarkAliasesPreparePass with `name()` so we don't have two places hardcoding the name?